smart-contracts/contracts/Unlock.sol

```diff
@@ -119,15 +130,19 @@ contract Unlock is
   }
 
   /**
-   * @notice Allows the owner to set the WETH token address.
-   * @dev Used for value calculations
+   * @notice Allows the owner to set key token addresses and configuration variables.
+   * @dev Used for value calculations and minting
    */
-  function initializeWeth(
-    address _weth
+  function configure(
+    address _udt,
+    address _weth,
+    uint _estimatedGasForPurchase
   ) public
     onlyOwner
   {
+    udt = IMintableERC20(_udt);
```

Comment:
Shouldn't this be ?
```
     udt = _udt;
```

Reply:
It's like this because I declared `udt` as that type instead of an address. It's really just a style question, the casting doesn't impact gas.